raster_api/runtime/src/app.py

```diff
@@ -61,20 +71,24 @@
     path_dependency=ItemPathParams,
     optional_headers=optional_headers,
     router_prefix="/stac",
-    gdal_config=settings.get_gdal_config(),
+    environment_dependency=settings.get_gdal_config,
     router=APIRouter(route_class=LoggerRouteHandler),
 )
 app.include_router(stac.router, tags=["Items"], prefix="/stac")
 
 cog = TilerFactory(
     router_prefix="/cog",
     optional_headers=optional_headers,
-    gdal_config=settings.get_gdal_config(),
+    environment_dependency=settings.get_gdal_config,
```

Comment:
I don't see us changing `ApiSettings.get_gdal_config` to be a property or anything here so I think we're not calling the function and hence not passing the auth info

Reply:
Isayah is saying this gets called downstream by `Depends`

Reply:
So `environment_dependency` is a callable in titiler. That was one of the breaking changes when `gdal_config` was removed.